testutil-server/src/main/java/io/spine/testing/server/blackbox/BlackBoxBoundedContext.java

```diff
@@ -61,7 +67,8 @@
  * effects.
  *
  * @param <T> the type of a sub-class for return type covariance
- * @apiNote The class provides factory methods for creation of different bounded contexts.
+ * @apiNote It is expected that instances of classes derived from {@code BlackBoxBoundedContext}
+ *          are obtained by factory methods provided by this class.
```

Comment:
Sometimes we put a period at the end of `@apiNote` and `@implNote` and sometimes we don't.

In JDK sources they have no periods. I suggest to follow this policy.

Reply:
As discussed vocally, such notes are usually a sentence or a paragraph. So periods apply.